src/common/Cargo.toml

```diff
@@ -38,6 +38,7 @@ easy-ext = "1"
 educe = "0.4"
 either = "1"
 enum-as-inner = "0.6"
+enumflags2 = { version = "0.7.8" }
```

Comment:
Why not use the `bitflags` crate which we already used?

Reply:
Oh, I was planning to use `bitflags` as described in that issue, but found `enumflags2` somewhere else which is more readable, easier to transform between `PbAction` and derive display along with each enum.

Reply:
I slightly prefer no to introduce new dependencies, in order to reduce the risk of supply chain security, and also reduce compilation time.

Not a requirement. You decide it.

Reply:
Then we have to write many ugly code as follows if using `bitflags`:
```rust
impl Display for AclMode {
    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
        if self == AclMode::Insert {
            write!(f, "a")
        }
        //....
    }
}

impl From<AclMode> for PbAction {
    fn from(mode: AclMode) -> Self {
        if mode == AclMode::Insert {
            return PbAction::Insert;
        }
        //....
    }
}
```
Let's introduce it this time. Not next time. 🥵